scripts/jjb-path-setup.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+from itertools import chain
+import os.path
+import re
+
+import click
+from sh import git, ErrorReturnCode_128
+import yaml
+
+
+@click.command()
+@click.option(
+    "--job-source",
+    "job_sources",
+    multiple=True,
+    default=[
+        "./",
+    ],
+    help=(
+        "Format URL;commitish, e.g. "
+        "https://github.com/rcbops/rpc-gating;master, or directory."
+    ),
+)
+@click.option(
+    "--update-default-job-sources",
+    "job_sources_updates",
+    multiple=True,
+    default=[],
+    help=(
+        "Allows for a subset of the job sources defined by `--job-sources to "
+        "be updated. Uses the same format as `--job-sources`."
+    ),
+)
+def setup(job_sources, job_sources_updates):
+    paths_by_dir = {}
+    for job_source in job_sources + job_sources_updates:
```

Comment:
I don't quite understand the need for sources and sources_updates as they appear interchangeable.